src/GraphQl/Mutations/ActionItemCategoryMutations.ts

```diff
@@ -0,0 +1,39 @@
+import gql from 'graphql-tag';
```

Comment:
Why create a separate file for these mutations ?

Reply:
I saw some separate files for some mutations and queries, and I thought we were moving towards more modular code, that's why😅. Shall I move them back into the main file?